controllers/topics/index.js

```diff
@@ -199,7 +199,7 @@ const createTopic = async (req, res) => {
       });
     }
 
-    const topic_id = await insertTopics([name]);
+    const topic_id = await insertSingleTopic(name);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line has been changed to use the new `insertSingleTopic` function. Ensure that the `followTopicViaCreateTopic` function can handle the new return type of `insertSingleTopic`. If `insertSingleTopic` returns a different type than `insertTopics`, you may need to adjust the `followTopicViaCreateTopic` function accordingly.

```diff
-    const topic_id = await insertTopics([name]);
+    const topic_id = await insertSingleTopic(name);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->